lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -464,6 +464,27 @@ sub update_status {
 
 =over 4
 
+=item get_status()
+
+Retrieve status of a job. Returns id, state, result, blocked_by_id.
+Preferrable over /job/<id> for performance and payload size, if you are only
+interested in the status.
+
+=back
+
+=cut
+
+sub get_status {
+    my ($self)    = @_;
```

Comment:
```suggestion
sub get_status ($self) {
```

Reply:
ok. should I add signatures to the whole module then or only this method?

Reply:
I just like to use them for new functions from the start. Of course you could convert the whole module if you like but that should then be a separate commit.